### ISSUE TITLE
some strange errors

### ISSUE THREAD

**Issue by TomasHurtz** [httpdispatch/MissedNotificationsReminder#29]
Hi there

Firstly - have been using your very useful app for some time on an older phone and it's worked great - thank you for your work (I would be happy to donate if there is a link available?)

I've recently got a new android 7.1.1 phone (and notice a couple issues - it's impossible to know if this issue is because of the new phone, or because of an updated app ?

Using v18 Debug.  Phone is in Priority Only mode.  I've given Override to app.

Settings:
- Notification is enabled
- Access to storage given

1) unable to add a custom notification sound (clicking on Show Advanced > Sound > Add notification - does nothing.  How can I a custom sound?  I have custom ringtones on the phone, but can't seem to access them from the app.

2) Battery Optimization - I go ahead and do this and I have to go to Battery Optimization > All apps > Scroll to Missed Notification Reminder > "Don't Optimize" > Done.  However, when the phone reboots, it reverts back to "not optimized"

3) When phone screen is off Notification doesn't work

Thanks for any help?

**Comment by httpdispatch** [comment]
1. That can be some security permission issue. What phone to do you have?
2. It is on OS level. On some phones you may also add application to autostart
3. There is an advanced option "Keep device awake" - you may try it. Also please use latest version from the play store such as debug version is not yet up to date. Do you use custom build?

**Comment by TomasHurtz** [comment]
1. it is vodafone smart V8
2. How to do that
3. I will try that

**Comment by httpdispatch** [comment]
2. Perhaps there are some additional security settings on the device

**Comment by httpdispatch** [comment]
Looks like your device has an extra battery settings https://userguide.vodafone.co.nz/vodafone/smart-v8-android-7-1/getting-started/extend-the-battery-life/

**Comment by TomasHurtz** [comment]
Yes, I turned that off.  Missed notification seems to be working OK now :-) 

Still unable to select a different reminder tone.  I think the phone won't discover the ringtones I added to media folder.  I will try with ringtone app.

Thanks for your help!
